django-verdant/rca/static/rca/css/components/hero.less

```diff
@@ -0,0 +1,130 @@
+.hero {
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: center;
+    margin: 0 -@spacing-one;
+    margin-bottom: @spacing-onepointfive;
+    padding: 0 @spacing-one;
+
+    @media @media-desktop-small {
+        margin-right: 0;
+        margin-left: 0;
+        padding: 0;
+    }
+
+    &__inner {
+        padding-bottom: @spacing-one;
+    }
+
+    &__text {
+        color: @color-white;
+        font-size: @font-size-24;
+        max-width: 745px;
+    }
+
+    &__title-wrapper {
+        display: flex;
+        justify-content: space-between;
+        align-items: flex-end;
+        margin-bottom: @spacing-one;
+    }
+
+    &__title {
+        color: @color-white;
+        font-size: 80px;
+        font-family: @font-family-knockout;
+        font-weight: @font-weight-knockout;
+        text-transform: uppercase;
+        line-height: 0.8;
+        margin: 0;
+    }
+
+    &__browse-link {
+        display: block;
+        color: @color-white;
+        font-family: @font-family-benton;
+        font-size: 15px;
+        text-transform: uppercase;
+        display: flex;
+        align-items: center;
+    }
+
+    &__browse-text {
+        max-width: 120px;
+        text-align: right;
+
+        &--text-alt {
+            color: lighten(@color-highlight-blue, 25%);
+        }
+    }
+
+    &__browse-icon {
+        width: 40px;
+        height: 40px;
+        fill: lighten(@color-highlight-blue, 25%);
+        margin-left: 10px;
+    }
+
+    &__search-wrapper {
+        position: relative;
+    }
+
+    &__search-input {
+        background: @color-white;
+        padding-right: 50px;
+
+        &::placeholder {
+            color: @color-highlight-blue;
+            font-family: @font-family-benton;
+        }
+    }
+
+    &__search-icon {
+        width: 22px;
+        height: 22px;
+        fill: @color-highlight-blue;
+        position: absolute;
+        right: 15px;
+        top: 50%;
+        transform: translateY(-50%);
+    }
+}
+
+.template-home-page {
+    .ui-menu {
```

Comment:
I'm happy with the decision of adding a components folder for new work. My slight worry with the ui-autocomplete styles being overridden here is that there are styles already in place for them in core.less etc. It's not that clear that these are being overridden here rather than newly declared. I can see they are scoped within the `.template-homepage` selector so they won't overwrite the other styles at least. I think perhaps the solution is a comment to make it clear that it's overriding the styles from the other files.